src/conspiracies/corpusprocessing/umap_hdb.py

```diff
@@ -290,7 +290,8 @@ def label_clusters(
 
 def embed_and_cluster(
     list_to_embed: List[str],
-    embedding_model: str = "vesteinn/DanskBERT",
+    language: str,
+    embedding_model: str,
```

Comment:
I would specify an interface (protocol) and then pass the model. If you then call it from the CLI I would create the model in the topmost layer an pass it down.

Reply:
That's not a bad idea since a similar logic is used in several places. The `umap_hdb` file which comes from Stine's branch is next in line for a refactoring in order to make it fit together more nicely with the rest of the pipeline and configuration options introduced by me.

Reply:
Hmm, I'll give it some thought and see how it might make sense in the mentioned upcoming refactor of `umap_hdb`.

Reply:
But I like the idea of thinking in the lines of dependency injection and maybe centralize model choices to one place in which one can then say to just use default Danish or English.